agent/src/beerocks/slave/tasks/topology_task.cpp

```diff
@@ -510,13 +513,49 @@ void TopologyTask::handle_topology_query(ieee1905_1::CmduMessageRx &cmdu_rx,
     m_btl_ctx.send_cmdu_to_broker(m_cmdu_tx, tlvf::mac_to_string(src_mac),
                                   tlvf::mac_to_string(db->bridge.mac));
 }
+
+bool TopologyTask::handle_vendor_specific(ieee1905_1::CmduMessageRx &cmdu_rx,
+                                          const sMacAddr &src_mac,
+                                          std::shared_ptr<beerocks_header> beerocks_header)
+{
+    if (!beerocks_header) {
+        LOG(ERROR) << "beerocks_header is nullptr";
+        return false;
+    }
+
+    // Since currently we handle only action_ops of action type "ACTION_BACKHAUL", use a single
+    // a single switch-case on "ACTION_BACKHAUL" only.
```

Comment:
```suggestion
    // Since currently we handle only action_ops of action type "ACTION_BACKHAUL", use
    // a single switch-case on "ACTION_BACKHAUL" only.
```
this looked like the reading tests you get on social media :-)

Reply:
Fixed.